src/backend/utils/init/miscinit.c

```diff
@@ -1779,6 +1804,8 @@ process_shared_preload_libraries(void)
 				   false);
 #endif
 
+	load_libraries_array();
```

Comment:
should before `load_libraries(shared_preload_libraries_string,
				   "shared_preload_libraries",
				   false);` ?

Reply:
This can be placed before or after. Placing it after loads the kernel’s dynamic library first, followed by loading the external dynamic library.